adage/analyze/api.py

```diff
@@ -5,7 +5,7 @@
 from organisms.api import OrganismResource
 from genes.api import GeneResource
 from tastypie import fields, http
-from tastypie.resources import Resource, ModelResource
+from tastypie.resources import Resource, ModelResource, convert_post_to_VERB
```

Comment:
Let me see if I can make an `api_test` test case do anything useful. I had thought about a test case for this but then thought we won't quite have the right testing infrastructure in place until we are doing E2E calls with a live server. That would obviously be the gold standard, but maybe there's something useful I can get out of `api_test`.

- [ ] see if an `api_test` test case would be helpful here